resources/views/admin.blade.php

```diff
@@ -146,22 +146,19 @@ function deleteEmployee(id){
                     });
                 });
 
-                let $id = null
 
                 //Edit Employee
+                let $id = null
                 $('#myTable tbody').on('click', 'button[id^="btn-edit"]', function (e) {
```

Comment:
access the button using `ID` instead of `element`

Reply:
okay noted

Reply:
resolved please check @Raza9798  #22